source/extensions/common/aws/signer_impl.h

```diff
@@ -71,9 +84,24 @@ class SignerImpl : public Signer, public Logger::Loggable<Logger::Id::http> {
                                         const std::map<std::string, std::string>& canonical_headers,
                                         absl::string_view signature) const;
 
+  std::vector<Matchers::StringMatcherPtr> defaultMatchers() {
+    std::vector<Matchers::StringMatcherPtr> default_excluded_headers{};
+    for (const auto& header : default_excluded_headers_) {
```

Comment:
The default_excluded_headers objects are really closely named.   Don't really have a suggestion here other than something that disambiguates these a bit more.

Reply:
renamed to `matcher_ptrs` to better reflect the return type